### DISCUSSION TITLE
Deployments Stuck at Building and Queued

### DISCUSSION THREAD

**Discussion by mingjunlu** [vercel/community#4936] [Help] [answered]
### Summary

Hi,

About half an hour ago, I pushed to GitHub but the deployment was not triggered.
So I tried pushing for a few more times. After a while, 4 deployments showed up.

One is stuck at `Building` for over 30 minutes, the other ones are still `Queued`.
Those 4 deployments couldn't be cancelled.

Here's part of the building message:

```
04:41:43 AM [build] Rearranging server assets...
04:41:43 AM [@astrojs/vercel/serverless] Bundling function ../../../../dist/entry.mjs
04:41:46 AM [build] Server built in 11.62s
04:41:46 AM [build] Complete!
Build Completed in /vercel/output [25s]
Deploying outputs...
An unexpected error happened when running this build. We have been notified of the problem. This may be a transient error. If the problem persists, please contact Vercel Support https://vercel.com/help
```

### Steps to Reproduce

1. Push to GitHub
2. Check the deployment status
3. Stuck at either `Building` or `Queued`

**Comment by mingjunlu** [comment]
[Update]

I successfully cancelled 3 of them, including the one stuck at `Building`.

Now there's one left, stuck at `Queued`. If I click to view its [deployment details](https://vercel.com/mingjunlu/personal-website/nDaGHjsUFBnYgoQQEa6gK5xjyKLi), it will show a 404 page.

**Comment by swarnava** [comment]
Hi,

You should be able to cancel deployment page from main page as well: 
<img width="1233" alt="image" src="https://github.com/vercel/community/assets/1162991/735bce08-ca69-4fc7-b4f1-6115c5d8dc2f">

We are investigating the 404 error.

**Reply by mingjunlu** [reply]
Hi @swarnava,

Thanks for the quick reply!

I tried to cancel the deployment like you suggest, unfortunately nothing happened.
Even the confirm modal didn't show up.

**Reply by swarnava** [reply, marked as answer]
It looks like the deployment was in bad state so we cancelled it. You should be able to access now or push new deployment.

**Reply by mingjunlu** [reply]
Thanks a bunch for sorting out the deployment issue! Things are back on track now. Much appreciated!
